GUI/widgets/plot_buttons.py

```diff
@@ -23,8 +26,18 @@ def initUI(self):
         self.example_button.setText("Example Button")
         self.example_button.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
 
+        self.plot_dropdown = QComboBox()
+        self.plot_dropdown.setEditable(True)
+        self.plot_dropdown.lineEdit().setAlignment(Qt.AlignCenter)
+        self.plot_dropdown.lineEdit().setReadOnly(True)
+        for plot_name in self.plot_names:
+            self.plot_dropdown.addItem(plot_name)
+        self.plot_dropdown.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1;")
```

Comment:
is it possible to set css of the dropdown from a separate file?

Reply:
Most likely, I was planning on batching all those changes as part of ticket 237